postgres-persistence/src/main/java/com/netflix/conductor/postgres/config/PostgresConfiguration.java

```diff
@@ -35,28 +36,38 @@
 @Import(DataSourceAutoConfiguration.class)
 public class PostgresConfiguration {
 
-    @Bean
-    public FlywayConfigurationCustomizer flywayConfigurationCustomizer() {
-        // override the default location.
-        return configuration -> configuration.locations("classpath:db/migration_postgres");
+    DataSource dataSource;
```

Comment:
Can you provide more details about this change and why its needed?

Reply:
We need this change because we cannot use the default flyway bean if we choose postgres as the external storage implementation. Before our addition, the flyway bean was created as a single instance and this class only had the path to the SQL commands to create the database. But if we use postgres external storage then we need to create 2 flyway beans and if we leave this class unchanged then Spring will return an error that the default bean cannot be created because a bean with the type Flyway already exists in the Application Context. Therefore, for correct co-existence and work of 2 different Flyway migrations, I had to manually create a bean also for this database.